test/ui_state/calibration/calibrate_temp_test.py

```diff
@@ -83,11 +83,12 @@ def test_loop(print_mock):
         [
             mock.call("Probe One", line=1),
             mock.call(
-                f"{(calibrate_temp.titrator.temp_probe_one.get_temperature()):0.3f}",
+                f"{(calibrate_temp.titrator.temperature_probe_control.get_temperature()):4.3f}",
```

Comment:
Why the format number change?

Reply:
I was going for consistency across all lcd calls that showed temperature. Both 0.3f and 4.3f were used in some places in the code so I went with 4.3f. I assume the actual number formatting will be decided by Dr. Onthank when the device is shown.